ansible/roles/ci/deploy/upload_debs/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Defaults aptly url
+
+aptly_url: http://10.10.10.62:8080
```

Comment:
my local aptly using the [docker image](https://hub.docker.com/r/shadowrobot/aptly/). We can update it when we have something deployed